src/traffic/core/flight.py

```diff
@@ -2589,6 +2592,119 @@ def intersects(  # type: ignore
         # given here for consistency in types
         ...
 
+    def get_toff_runway(
```

Comment:
can we reuse `takeoff_from_runway` but pass an argument (string) with the method which can come to this one? would be more helpful to understand why we need two methods

Reply:
I find it a bit confusing because it does not return an iterable while takeoff_from_runway does